storybook/storybook-css/src/components/Breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Breadcrumb } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/src/breadcrumb/breadcrumb.scss'
+
+const meta = {
+  title: 'CSS Components/Breadcrumb',
+  component: Breadcrumb,
+} satisfies Meta<typeof Breadcrumb>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const BreadcrumbStory: Story = {
+  render: (args) => (
+    <Breadcrumb {...args}>
+      <Breadcrumb.Item href="/some-url.com">home</Breadcrumb.Item>
```

Comment:
Maybe we should link to the same page, so you don't accidentally leave storybook

```suggestion
      <Breadcrumb.Item href="#">home</Breadcrumb.Item>
```

Reply:
done